HelperApp/src/main/java/com/cdw/springenablement/helper_App/services/AdminServiceImpl.java

```diff
@@ -123,36 +128,39 @@ public void removeResident(Integer residentId) throws Exception {
             if (isResident) {
                 userRepository.deleteById(Long.valueOf(residentId));
             } else {
-                throw  new Exception("cannot delete ..not a resident");
+                throw  new HelperAppException("cannot delete ..not a resident");
             }
         } else {
-            System.out.println("User not found.");
+           throw  new HelperAppException("User Not found");
         }
     }
 
+    /**
+     *
+     * @param helperId
+     * Method that remove the helper
+     */
     @Override
-    public void removeHelper(Integer helperId) throws Exception {
+    public void removeHelper(Integer helperId)  {
         Helper helper = helperRepository.findById(Long.valueOf(helperId)).orElse(null);
         if (helper == null) {
```

Comment:
try to avoid conditions on null checks and use internal ! flags to check conditions